openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py

```diff
@@ -72,6 +72,9 @@ def get_record_data(self, data):
     @staticmethod
     def _get_image_type(image_data):
         # Returns the image url and key for the image in `image_data` dict.
```

Comment:
The function name is confusing if it's doing what the comment says 🤔 By the tests looks like it's actually _not_ returning the image type/file extension but a bunch of other stuff.

Can we update the name? Maybe not in this PR but letting the note.

Reply:
Good point! I think the original function name must have referred to the fact that we're selecting an image from the `image_data` dict, _by type_ (web/print/full)? 😮 That's _definitely_ confusing. Updated!